packages/react-core/src/components/Select/Select.tsx

```diff
@@ -1250,7 +1254,7 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
         ref={this.parentRef}
         {...getOUIAProps(Select.displayName, ouiaId !== undefined ? ouiaId : this.state.ouiaStateId, ouiaSafe)}
         {...(width && { style: { width } })}
-        {...(validated !== ValidatedOptions.default && { 'aria-describedby': ariaDescribedby })}
+        {...(ariaDescribedby && { 'aria-describedby': ariaDescribedby })}
```

Comment:
I believe if this is moved to the `<SelectToggle>` on line 1260, the attribute would apply to the correct element.